frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -300,17 +300,24 @@ const ProjectSelector = ({ onError }: ProjectSelectorProps) => {
     setCustomProject(event.target.value);
   };
 
+  const { updateRefreshRate } = useRefreshUpdater();
+
   // Autorefresh will eventually be passed via hooks to the cards to enforce their update intervals
   const [autoRefresh, setAutoRefresh] = React.useState<boolean>(true);
-
+  const REFRESH_RATE_MS = 30000;
```

Comment:
should we read this value in from the context since it will have a default value? This would also mean that the project selector wouldn't overwrite the refresh rate when auto refresh / state change.

Reply:
I guess my line of thinking was that because this is setting the context value, it wouldn't need to read it as well. I'll update it so that it initializes based off the read context value.

Reply:
hm actually not sure why when I try it like this it doesn't work, or how this would work.

Reply:
actually I think I figured it out. awesome!